README.md

```diff
@@ -80,8 +80,10 @@ Within GitLab you will have the following repo structure:
     gcloud services enable cloudbuild.googleapis.com
     gcloud services enable anthos.googleapis.com
     gcloud services enable serviceusage.googleapis.com
+    gcloud services enable binaryauthorization.googleapis.com
```

Comment:
@bgood -- can you check the "testing" install script to make sure they are installed there as well?